src/Raven.Server/Documents/Handlers/Debugging/IdentityDebugHandler.cs

```diff
@@ -37,5 +38,55 @@ public Task GetIdentities()
 
             return Task.CompletedTask;
         }
+
+
+        [RavenAction("/databases/*/identity/next", "GET", AuthorizationStatus.ValidUser, IsDebugInformationEndpoint = true)]
+        public async Task NextIdentityFor()
+        {
+            var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
+
+            if (name[name.Length - 1] != '|')
+                name += '|';
+            
+            (var clusterEtag, var clusterId) = await Database.ServerStore.GenerateClusterIdentityAsync(name, Database.Name);
```

Comment:
What is `ClusterEtag` and `ClusterId` ?
It should be the `Index`, not etag and the new identity value.